src/fetch_agent/fetch_agent/agent.py

```diff
@@ -0,0 +1,58 @@
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import String
+
+from uagents import Agent, Context
+from uagents.setup import fund_agent_if_low
+
+from aca_protocols.station_query_protocol import (
+    StationQueryRequest,
+    StationQueryResponse,
+)
+
+from aca_protocols.acs_registry_id import acs_id
+
+agent = Agent(
+    name="car",
+    seed="Car1",
+    port=8002,
```

Comment:
In future this will be unnecessary bc these clients will be running on separate machines. Is that true or am I tweaking?

Reply:
yea, we will need to look into how to manage the ports and addresses since they won't run on the same host so localhost won't work.